Content.Server/PDA/PdaSystem.cs

```diff
@@ -55,9 +55,23 @@ public override void Initialize()
             SubscribeLocalEvent<PdaComponent, CartridgeLoaderNotificationSentEvent>(OnNotification);
 
             SubscribeLocalEvent<StationRenamedEvent>(OnStationRenamed);
+            SubscribeLocalEvent<EntityRenamedEvent>(OnEntityRenamed);
             SubscribeLocalEvent<AlertLevelChangedEvent>(OnAlertLevelChanged);
         }
 
+        private void OnEntityRenamed(ref EntityRenamedEvent ev)
+        {
+            var query = EntityQueryEnumerator<PdaComponent>();
+
+            while (query.MoveNext(out var uid, out var comp))
+            {
+                if (comp.OwnerName == ev.OldName)
+                {
+                    SetOwner(uid, comp, ev.NewName);
+                }
+            }
+        }
+
```

Comment:
Won't this get cooked if 2 people have the same name?

Reply:
It will. I should've looked at the rename command code closer before moving it.